clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/provider/KubernetesProviderConfig.groovy

```diff
@@ -45,11 +47,13 @@ class KubernetesProviderConfig implements Runnable {
   @DependsOn('kubernetesNamedAccountCredentials')
   KubernetesProvider kubernetesProvider(KubernetesCloudProvider kubernetesCloudProvider,
                                         AccountCredentialsRepository accountCredentialsRepository,
-                                        KubernetesV1CachingAgentDispatcher kubernetesV1CachingAgentDispatcher) {
+                                        KubernetesV1CachingAgentDispatcher kubernetesV1CachingAgentDispatcher,
+                                        KubernetesV2CachingAgentDispatcher kubernetesV2CachingAgentDispatcher) {
```

Comment:
Would like to see this as a `Collection` if we add another track.

Reply:
:+1: